src/smart-components/ChannelSettings/components/UserListItem/index.tsx

```diff
@@ -11,8 +11,6 @@ import ContextMenu, { MenuItems } from '../../../../ui/ContextMenu';
 
 import './user-list-item.scss';
 
-const COMPONENT_NAME = 'sendbird-user-list-item--small';
```

Comment:
wasnt it better this way :)

Reply:
I thought this way is better, but it lets me debug very hard

Reply:
okok